File: Iniciante/URI 1002.cpp
#include <iostream>
#include <stdio.h>
#include <math.h>

using namespace std;

int main()
{

  double raio, pi, area;

  pi = 3.14159;

  cin >> raio;

  area = pi * (raio * raio);

  printf("A=%.4lf\n", area);

  return 0;
}

File: Iniciante/URI 1002.cs
using System; 

class URI {

    public static void Main(string[] args) { 
        
        double pi = 3.14159;

        double raio, area;
        
        raio = Double.Parse(Console.ReadLine());
        
        area = pi * Math.Pow(raio, 2);
        
        Console.WriteLine("A={0:F4}", area);
        
    }
}